amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/exceptions/LeasePendingDeletion.java

```diff
@@ -32,17 +32,18 @@
  * Helper class for cleaning up leases.
  */
 @Accessors(fluent = true)
+@EqualsAndHashCode
 @Value
-@EqualsAndHashCode(exclude = {"queueEntryTime"})
 public class LeasePendingDeletion {
-    private final StreamIdentifier streamIdentifier;
-    private final Lease lease;
-    private final ShardInfo shardInfo;
-    private final ShardDetector shardDetector;
+
+    StreamIdentifier streamIdentifier;
```

Comment:
Are these changes intended ?

Reply:
Yes. Per Lombok [@Value](https://projectlombok.org/features/Value):
> all fields are made private and final by default, and setters are not generated

Explicit `private final` are unnecessary/redundant, and get flagged by IntelliJ as such.